lib/segment/src/index/field_index/full_text_index/inverted_index/mod.rs

```diff
@@ -47,7 +47,13 @@ impl TokenSet {
         self.0.binary_search(token).is_ok()
     }
 
+    /// Checks if the current set contains all given tokens.
+    ///
+    /// Returns false if the subset is empty
     pub fn has_subset(&self, subset: &TokenSet) -> bool {
+        if subset.is_empty() {
+            return false;
+        }
```

Comment:
Is this required? It seems counter intuitive to me, and it doesn't look like it's necessary for the callers.